.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Type: Bug'
+assignees: ''
+---
+
+## Bug Description
+
+[Provide a clear and concise description of the bug.]
+
+## Steps to Reproduce
+
+[Describe the steps to reproduce the bug in a detailed manner.]
+
+1. Step 1
+2. Step 2
+3. ...
+
+## Expected Behavior
+
+[Describe what you expected to happen.]
+
+## Current Behavior
+
+[Describe what is currently happening instead.]
+
+## Command Used
+
+[Include the exact command(s) you used when encountering the bug.]
+
+## Screenshots or Output
+
+[If applicable, include screenshots or the output of the CLI tool that helps illustrate the issue.]
+
+## Environment
+
+- Operating System: [Specify the operating system you're using]
+- Basti Version/Commit: [Specify the version or commit of the tool]
+- Node Version: [Specify the version of Node.js you're using]
+- Package Manager: [Specify the package manager you're using, e.g. npm or yarn]
+- Package Manager Version: [Specify the version of the package manager you're using]
+
+## Additional Information
```

Comment:
I think this field might be removed to make the template shorter. People will provide everything they consider important in other fields (Hopefully :D )